static/app/locale.tsx

```diff
@@ -72,7 +72,6 @@ function getClient(): Jed | null {
     // If this happens, it could mean that an import was added/changed where
     // locale initialization does not happen soon enough.
     const warning = new Error('Locale not set, defaulting to English');
-    console.error(warning); // eslint-disable-line no-console
```

Comment:
I tried setting locale in test setup files, but it was still erroring in certain tests. Removing because it was kind of useless anyway (and we're capturing in Sentry).

Reply:
oh my god 🙏